src/components/Registration/BusinessLookup.vue

```diff
@@ -7,6 +7,7 @@
         autocomplete="chrome-off"
         class="mt-5"
         append-icon=""
+        :menu-props="{ maxHeight: 380 }"
```

Comment:
This may not need the binding indicator (:) because it's not a reactive value. You may be able to simplify.